src/androidTest/java/de/dennisguse/opentracks/util/StringUtilsTest.java

```diff
@@ -184,4 +200,105 @@ public void testFormatSpeed() {
         assertEquals("12.0 km/h", StringUtils.formatSpeed(context, Speed.of(3.34), true, true));
         assertEquals("7.5 mph", StringUtils.formatSpeed(context, Speed.of(3.34), false, true));
     }
+
+    @Test
+    public void testFormatDateTime_enUs() {
+        // give
+        LocalDate yesterday = LocalDateTime.ofInstant(Instant.now(), ZoneId.systemDefault()).toLocalDate().minus(1, ChronoUnit.DAYS);
```

Comment:
Why not use artificial Instants?
Something like `Instant.parse("2020-02-02T02:02:21Z")`?

Reply:
Much better.